src/macports1.0/reclaim.tcl

```diff
@@ -527,20 +521,15 @@ namespace eval reclaim {
 
         } else {
 
-            ui_msg "Found unrequested ports without requested dependents: [join $unnecessary_names {, }]."
-            if {[info exists macports::ui_options(questions_multichoice)]} {
-                set retstring [$macports::ui_options(questions_multichoice) "Would you like to uninstall these ports?" "" $unnecessary_names]
-
-                if {[llength $retstring] > 0} {
-                    foreach i $retstring {
-                        set name [lindex $unnecessary_names $i]
-                        set port $ports($name)
-
-                        ui_msg "Uninstalling: $name"
+            ui_msg " [join $unnecessary_names {, }]."
```

Comment:
This seems like a very unreadable way to print the ports. Did you test that this will actually stay readable with many ports?

Reply:
This is actually an error, that line should go.